packages/near-api-js/src/account.ts

```diff
@@ -483,40 +483,17 @@ export class Account {
      * @param contractId NEAR account where the contract is deployed
```

Comment:
This JSDoc is configured for separate arguments, now that `viewFunction` expects a single object argument I believe we need something similar to the previous `options` - https://jsdoc.app/tags-param.html